doc/fr/faq/remote_server.rst

```diff
@@ -334,13 +334,7 @@ central.
 
 3. Fichiers manquants dans le répertoire /etc/centreon
 
-Vérifiez que le répertoire **/etc/centreon** contient les fichiers suivants :
-
-* instCentCore.conf
-* instCentPlugins.conf
-* instCentWeb.conf
-
-et que ces fichiers ne sont pas vides, sinon copiez les depuis le serveur Centreon
+Vérifiez que le fichier **/etc/centreon/instCentWeb.conf** n'est pas vide, sinon copiez le depuis le serveur Centreon
```

Comment:
why this file is needed, when you seem to don't use it anymore ? 
https://github.com/centreon/centreon/pull/8172/files#diff-92e7108e0f126f93ae0f2687f4380febL1738